src/gatewayBFF.ts

```diff
@@ -365,6 +365,22 @@ export class GatewayBFF {
      */
     private addConfigurationRoute(cb: Function) {
         this.server.addRoute('/', HTTP_METHODS.GET, (req, res) => {
+
+            if(req.query.fragment) {
+                const fragment = this.exposedConfig.fragments[req.query.fragment];
+                if(fragment) {
```

Comment:
`      if(!fragment) return res.status(404).end();
        return res.status(200).json({
                        assets: fragment.assets,
                        dependencies: fragment.dependencies,
                        version: fragment.version
                    });
                
`

Reply:
'else' block is unnecessary, I will remove that but I think moving falsy control to the top and removing brackets will not affect readability. I prefer brackets even it is one line for debugging purposes.